src/microsoft-trydotnet-editor/package.json

```diff
@@ -50,13 +50,14 @@
     "ts-mocha": "10.0.0",
     "ts-node": "10.8.1",
     "typescript": "4.7.4",
-    "webpack": "5.73.0",
+    "webpack": "^5.90.3",
```

Comment:
From what I know, we have been using fixed versions for compliance reasons. We'd likely want to preserve that here as well.

Reply:
Cannot revert this change to 5.73.0 because is a security vulnerability in webpack in the range of 5.0.0 - 5.75.0